critiquebrainz/db/review.py

```diff
@@ -591,6 +591,19 @@ def get_popular(limit=None):
     shuffle(reviews)
     return reviews[:limit]
 
+def get_top_languages():
+    languages = {}
+    supported_languages_iso639 = []
+    top_languages_iso639 = []
+    for lang in list(pycountry.languages):
+        if 'iso639_1_code' in dir(lang):
+            supported_languages_iso639.append(lang.iso639_1_code)
+    for langcode in supported_languages_iso639:
+          review = list_reviews(language = str(langcode))
```

Comment:
It would be really great if you could use a SQL query for fetching the top reviews. This code makes a lot of queries to the database for fetching the count of all the languages. However, we can do fetch the top languages in a single query as (selecting the top 6 most used languages): 


`SELECT language, 
                count(id) as count 
    FROM review 
GROUP BY language 
ORDER BY count desc limit 6`

So, I would suggest that you implement fetching the top languages using the above query. Make sure you use `sqlalchemy` like other functions of `review.py`. Also make sure to follow guidelines for writing SQL queries (https://github.com/metabrainz/guidelines/blob/master/Python.md#sql)